noetic/distribution.yaml

```diff
@@ -808,7 +808,7 @@ repositories:
       tags:
         release: release/noetic/{package}/{version}
       url: https://github.com/ros-gbp/octomap-release.git
-      version: 1.9.5-2
+      version: 1.9.5-3
```

Comment:
I only see [`1.9.5-2` in the release repo](https://github.com/ros-gbp/octomap-release#octomap-noetic---195-2). Did an error occur during the release?

Reply:
I used `git-bloom-release` instead of `bloom-release`. The release tags are there: https://github.com/ros-gbp/octomap-release/releases. I reran it and will update the release number. I believe `git-bloom-release` does not update the README, only the release number in `tracks.yaml`.